orbyter-dl-dev/README.md

```diff
@@ -36,52 +77,55 @@ the useful packages for ML development.
 
 System:
 
-* Ubuntu 18.04.3 LTS
-* Python 3.7.3
+* Ubuntu 20.04 LTS
+* Python 3.8.5
 
 ## Included Packages :
 
-* bayesian-optimization==1.0.1
+* bayesian-optimization==1.2.0
 * black==19.3b0
-* bokeh==1.3.4
-* boto3==1.9.233
-* Click==7.0
-* coloredlogs==10.0
-* dask[complete]==2.4.0
-* dask-ml==1.0.0
-* dlib==19.17.0
-* fastai==1.0.57
+* bokeh==2.2.3
+* boto3==1.16.60
+* Click==7.1.2
+* coloredlogs==15.0
+* dask[complete]==2021.1.1
+* dask-ml==1.7.0
+* dlib==19.21.1
+* dvc==1.11.13
+* fastai==2.2.5
 * Keras==2.3.0
-* fire==0.2.1
-* flake8==3.7.8
-* ipdb==0.12.2
-* isort==4.3.21
-* jsonlines==1.2.0
+* fire==0.4.0
+* flake8==3.8.4
+* ipdb==0.13.4
+* isort==5.7.0
```

Comment:
@anishpdalal Can we port the jupyterlab autocompletion fix that @joegoldbeck put in the ml-dev image here as well? We just need to pin `ipython` to a particular version.. Otw autocompletion for Jupyter breaks.